qemu/block.go

```diff
@@ -173,14 +174,14 @@ func (bd BlockDevice) Commit(d *Domain, overlay string, timeout time.Duration) e
 	})
 }
 
-// CancelJob cancels any active block jobs on a block device.
+// CancelJob cancels a active block job.
```

Comment:
```suggestion
// CancelJob cancels an active block job.
```

Reply:
I care less about the `a` `an` thing, more about the function name (it was renamed to `Cancel()`, no?